config/schedule.rb

```diff
@@ -0,0 +1,32 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Example:
+#
+# set :output, "/path/to/my/cron_log.log"
+#
+# every 2.hours do
+#   command "/usr/bin/some_great_command"
+#   runner "MyModel.some_method"
+#   rake "some:great:rake:task"
+# end
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
+
+# Learn more: http://github.com/javan/whenever
+
+env :GEM_PATH, '/usr/local/bundle' # defines where to find rake command
+set :output, '/var/log/cron.log' # log location
+
+# We override rake job type, as we don't want envrinoment specific task
+job_type :rake, "cd :path && bundle exec rake :task --silent :output"
+
+every 1.hour do # 1.minute 1.day 1.week 1.month 1.year is also supported
+  # the following tasks are run in parallel (not in sequence)
+  # rake "my:rake:task"
+  rake "notify:send_noko_format_warning['pecas_dev']"
```

Comment:
One issue - need to work out how to handle this in "not prod" - the task running in Docker will import entries live every hour and alert users.

Currently I have the cron calling the pecas_dev Slack group (which is just me) - we'll need to change that in prod to 'ombulabs'... but then anyone running the app locally with an active Slack OAuth will start sending duplicate messages.

Maybe it's just an env var? Any idea for a better way would be appreciated

Reply:
Maybe the cron job should not be run inside the container, we should be able to run the   rake `notify:send_noko_format_warning['pecas_dev']` task using the container and we configure the cron job where we host the app to run that rake task, I think Pecas is running in Heroku so we could add a cron dyno to call   `rake "notify:send_noko_format_warning"` with whatever cron config we want from there, let me know what you think